lib/sinon/behavior.js

```diff
@@ -101,7 +101,7 @@ function callCallback(behavior, args) {
 
 var proto = {
     create: function create(stub) {
-        var behavior = extend({}, proto);
+        var behavior = extend(stub.bind(), proto);
```

Comment:
what does this do? I am guessing this is an easy way of wrapping the stub in a closure so you can change the copy instead of the original. am I right?

Reply:
yes. We could replace it with 

```js
var behavior = extend(function(){
   var args = Array.prototype.slice.call(arguments);
   return stub.apply(null, args);
}, proto);
```

But the .bind version is more concise.